src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -729,9 +729,17 @@ void OrbitMainWindow::OnNewSelectionBottomUpView(
 }
 
 std::string OrbitMainWindow::OnGetSaveFileName(const std::string& extension) {
-  std::string filename =
-      QFileDialog::getSaveFileName(this, "Specify a file to save...", nullptr, extension.c_str())
-          .toStdString();
+  QFileDialog dialog(this);
+  dialog.setFileMode(QFileDialog::FileMode::AnyFile);
+  dialog.setAcceptMode(QFileDialog::AcceptMode::AcceptSave);
+  dialog.setNameFilter((extension + " (*" + extension + ")").c_str());
```

Comment:
We should use QString::fromStdString instead of std::string::c_str since the former is more efficient and safer.

Reply:
And you can also consider to use format strings like `QString{"%1 (*%1)"}.arg(QString::fromStdString(extension))`. Looks slightly more readable to me, but I don't feel strongly about it.

Reply:
Changed, thanks. I've not updated this to use the format string though, I actually prefer the other one readability-wise.